### PR TITLE
E6 rewrite

### PR DESCRIPTION
**Pull request by martinj**

### CODE REVIEW

**Comment by gunnarlium** [comment]
To make the build pass on Node 4, you need to depend on `eslint-config-yandex` explicitly.

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 language: node_js
 
 node_js:
-  - 0.12
-  - 4.0
+  - 4
+  - 6
```
Comment (gunnarlium) [inline .travis.yml:5]: I think these shold reflect the node versions we are actually using, ie. be more explicit.

Comment (martinj) [reply .travis.yml:5]: Since this is a module i rather not lock it down to a specific version. The specifity should rather be in the applications.

**Comment by martinj** [comment]
Updated. Fixed broken travis build

**Pull request merged**